bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableConfiguration.java

```diff
@@ -178,6 +178,13 @@ public BigtableOptions toBigtableOptions() throws IOException {
    */
   public Configuration toHBaseConfig() {
     Configuration config = new Configuration();
+
```

Comment:
I wanted to turn off this behavior for Dataflow by default, but keep it on everywhere else.